src/tiny_slam/tiny_fascade.h

```diff
@@ -9,20 +9,45 @@
 
 #include "tiny_world.h"
 
+/*!
+ * \brief Derived class from LaserScanObserver to manage all world
+ *
+ * Handles data came from laser scanner and locates views if he is set.
+ * Presents a pattern fascade for all algorithm "tiny slam"
```

Comment:
Better "connects internal framework's components to a single tinySLAM method (the fascade pattern is applied)"